tests/common/platform/device_utils.py

```diff
@@ -897,7 +897,7 @@ def post_reboot_analysis(marker, event_counters=None, reboot_oper=None, log_dir=
             summary_file_name = request.node.name + "_summary.json"
 
         report_file_dir = os.path.realpath((os.path.join(os.path.dirname(__file__),
-                                           "../logs/platform_tests/")))
+                                           "../../logs/platform_tests/")))
```

Comment:
Non-blocking comment: I wonder if there's some common variable that we can use to get to the tests directory...

Reply:
Yeah, it's not ideal. This style is used quite heavily throughout the codebase though. I'd suggest a more widespread refactoring of this style if deemed worth it.